CICD.Tools.ValidatorErrorsToMarkdown/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Microsoft.Build.Locator;
+using Microsoft.CodeAnalysis.MSBuild;
+
+using Skyline.DataMiner.CICD.FileSystem;
+
+namespace Skyline.DataMiner.CICD.Tools.ValidatorErrorsToMarkdown
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        public static async Task<int> Main(string[] args)
+        {
+            var fileLocation = new Option<string>(
+                name: "--fileLocation",
+                description: "File containing the ErrorMessages.")
+            {
+                IsRequired = true
+            };
+
+            var resultLocation = new Option<string>(
+                name: "--resultLocation",
```

Comment:
```suggestion
                name: "--outputDirectoryPath",
```

we'll be generating several markdown files. So likely we'll want to ask for a directory path